services/github/src/utils.js

```diff
@@ -59,18 +61,37 @@ throttleRequest.request = request;
  * @returns boolean
  */
 const shouldSkipCommit = ({ commits, head_commit = {} }) => {
-  const testRe = new RegExp('\\[(skip ci|ci skip)\\]', 'i');
-
   let last_commit = head_commit && head_commit.message ? head_commit : false;
 
   if (!last_commit && Array.isArray(commits) && commits.length > 0) {
     last_commit = commits[commits.length - 1];
   }
 
-  return last_commit && testRe.test(last_commit.message);
+  return last_commit && ciSkipRegexp.test(last_commit.message);
+};
+
+module.exports = {
+  throttleRequest,
+  shouldSkipCommit,
+};
+
+/**
+ * Check if pull_request event should be skipped.
+ * It can happen when pull request contains keywords in it's title:
+ * "[skip ci]" or "[ci skip]"
+ *
+ * @param {body} object event body
+ * @param {body.pull_request} object[]
+ * @param {body.pull_request.title} string
+ *
+ * @returns boolean
+ */
+const shouldSkipPullRequest = ({ pull_request }) => {
+  return pull_request !== undefined && ciSkipRegexp.test(pull_request.title);
```

Comment:
I think it's more common for people to use the description for that?

Reply:
I'm not sure, but we can have both

Reply:
Formalized this in the RFC.. should have done it initially: <https://github.com/taskcluster/taskcluster-rfcs/pull/177>